Aztec/Classes/TextKit/ParagraphProperty/Header.swift

```diff
@@ -98,7 +98,7 @@ open class Header: ParagraphProperty {
 //
 private extension Header {
     struct Constants {
-        static let defaultFontSize = Float(14)
+        static let defaultFontSize = Float(16)
```

Comment:
the old value of default font size corresponded to h5 so I updated this as h5's new value also